app/consapp/convbin/convbin.c

```diff
@@ -58,70 +58,70 @@
-    "     -span span   time span (h) [all]",
-    "     -r format    log format type:",
-    "                  sbp       - Swift Navigation SBP",
-    "                  json      - Swift Navigation SBP-JSON",
-    "     -ro opt,opt  receiver option(s) (use comma between multiple opt):",
-    "                  CONVBASE  - convert base station observations",
-    "                  BASEPOS   - determine base station location from input file",
-    "                  EPHALL    - include all ephemeris",
-    "                  INVCP     - invert carrier phase polarity",
-    "                  OBSALL    - include observations with RAIM flag set",
-    "     -f freq      number of frequencies [2]",
-    "     -hc comment  rinex header: comment line",
-    "     -hm marker   rinex header: marker name",
-    "     -hn markno   rinex header: marker number",
-    "     -ht marktype rinex header: marker type",
-    "     -ho observ   rinex header: oberver name and agency separated by /",
-    "     -hr rec      rinex header: receiver number, type and version separated by /",
-    "     -ha ant      rinex header: antenna number and type separated by /",
-    "     -hp pos      rinex header: approx position x/y/z separated by /",
-    "     -hd delta    rinex header: antenna delta h/e/n separated by /",
-    "     -v ver       rinex version [3.00]",
-    "     -oi          include iono correction in rinex nav header [off]",
-    "     -ot          include time correction in rinex nav header [off]",
-    "     -ol          include leap seconds in rinex nav header [off]",
-    "     -halfc       half-cycle ambiguity correction [off]",
-    "     -mask   [sig[,...]] signal mask(s) (sig={G|R|E|J|S|C|I}L{1C|1P|1W|...})",
-    "     -nomask [sig[,...]] signal no mask (same as above)",
-    "     -x sat       exclude satellite",
-    "     -y sys       exclude systems (G:GPS,R:GLO,E:GAL,J:QZS,S:SBS,C:BDS,I:IRN)",
-    "     -d dir       output directory [same as input file]",
-    "     -c staid     use RINEX file name convention with staid [off]",
-    "     -o ofile     output RINEX OBS file",
-    "     -n nfile     output RINEX NAV file",
-    "     -g gfile     output RINEX GNAV file",
-    "     -h hfile     output RINEX HNAV file",
-    "     -q qfile     output RINEX QNAV file",
-    "     -l lfile     output RINEX LNAV file",
-    "     -s sfile     output SBAS message file",
-    "     -trace level output trace level [off]",
-    "",
-    " If not any output file is specified, default output files <file>.obs,",
-    " <file>.nav, <file>.gnav, <file>.hnav, <file>.qnav, <file>.lnav and",
-    " <file>.sbs are used. Empty output files are deleted after processing.",
-    "",
-    " If log format type is not specified, format type is recognized by the input",
-    " file extension as following:",
-    "     *.sbp        Swift Navigation SBP",
-    "     *.json       Swift Navigation SBP-JSON",
+"",
```

Comment:
nit, what format should we follow in here. It seems `rtkrcv` uses indented format, but `rnx2rtkp` doesn't.

Reply:
This is the indentation used upstream